### ISSUE TITLE
ChoiceList is not work with component library mode

### ISSUE THREAD

**Issue by nguyenvanduocit** [ownego/polaris-vue#402]
When import ChoiceList as a component like the example bellow:

```
<script lang="ts" setup>
import { ChoiceList } from '@ownego/polaris-vue'
</script>

<template>
<ChoiceList>...</ChoiceList>
</template>
```

nothing happen.

But it work when use as a plugin.

Can you pls take sometime on this?

**Comment by haild173094** [comment]
Hi @nguyenvanduocit ,
I created a new project and used the import ChoiceList as component, and it worked without any issues. Can you please provide more details about your code or share the error log from your console if it exists? I'd be happy to take a look and help you troubleshoot.
![image](https://github.com/user-attachments/assets/dfef9b0a-4c0b-47d9-845c-5480257d88cc)
```
<script setup>
import { ref } from 'vue';
import { ChoiceList, TextField } from '@ownego/polaris-vue';

const selected = ref(['hidden']);
</script>

<template>
    <ChoiceList
      title="Visibility"
      :choices="[
        {label: 'Hidden', value: 'hidden'},
        {label: 'Optional', value: 'optional'},
        {label: 'Required', value: 'required'},
      ]"
      v-model="selected"
    />
    <TextField
      v-model = "selected"
    >
    </TextField>
</template>
```

**Comment by blacklen** [comment]
Hi, this issue has been resolved in version [v2.1.11](https://github.com/ownego/polaris-vue/releases/tag/v2.1.11)

**Comment by nguyenvanduocit** [comment]
thanks
